cypress/integration/interpretations.cy.js

```diff
@@ -0,0 +1,169 @@
+
+        // it's possible to write a new interpretation text
+        cy.getBySel('interpretation-form')
+            .find(`input[placeholder="${TEST_WRITE_INTERPRETATION_LABEL}"]`)
+            .click()
+
+        typeTextarea('interpretation-form', TEST_INTERPRETATION_TEXT)
+
+        // the new interpretation can be saved and shows up in the list
+        cy.getBySel('interpretation-form')
+            .contains(TEST_POST_INTERPRETATION_LABEL)
+            .click()
+
+        cy.getBySel('interpretations-list').should(
+            'contain',
+            TEST_INTERPRETATION_TEXT
+        )
+        cy.getBySel('interpretations-list').should(
+            'contain',
+            'See interpretation'
+        )
+    })
+
+    it('the new interpretation can be edited', () => {
+        cy.getBySel('interpretation-edit-button').first().click()
+
+        cy.getBySel('interpretations-list').should('contain', 'Update')
+        cy.getBySel('interpretations-list').should('contain', TEST_CANCEL_LABEL)
+
+        clearTextarea('interpretations-list')
+        typeTextarea('interpretations-list', TEST_INTERPRETATION_TEXT_EDITED)
+
+        cy.getBySel('interpretations-list').contains('Update').click()
+
+        expectInterpretationFormToBeVisible()
+
+        cy.getBySel('interpretations-list').should(
+            'contain',
+            TEST_INTERPRETATION_TEXT_EDITED
+        )
+    })
+
+    it('the new interpretation can be viewed in the modal and interacted with', () => {
+        cy.getBySel('interpretations-list')
+            .contains('See interpretation')
+            .click()
+        cy.getBySel('interpretation-modal').should(
+            'contain',
+            'Viewing interpretation:'
+        )
+        cy.getBySel('interpretation-modal').should(
+            'contain',
+            TEST_INTERPRETATION_TEXT_EDITED
+        )
+
+        // it's possible to add a comment to the new interpretation
+        expectInterpretationThreadToBeVisible()
+
+        cy.getBySel('interpretation-modal')
+            .find(`input[placeholder="${TEST_WRITE_REPLY_LABEL}"]`)
+            .click()
+
+        typeTextarea('interpretation-modal', TEST_INTERPRETATION_COMMENT_TEXT)
+
+        // the comment can be saved and shows up in the interpretation thread
+        cy.getBySel('interpretation-modal').contains('Post reply').click()
+
+        cy.getBySel('interpretation-modal').should(
+            'contain',
+            TEST_INTERPRETATION_COMMENT_TEXT
+        )
+
+        // the page size can be changed
+        // (change page size for testing that the sorting does not apply to the main view)
+        cy.getBySel('interpretation-modal').contains('Rows per page').click()
+        cy.getBySel('dhis2-uicore-select-menu-menuwrapper')
+            .contains('5')
+            .click()
+        cy.getBySel('interpretation-modal')
+            .find('[data-test="dhis2-uiwidgets-pagination-pagesize-select"]')
+            .should('contain', '5')
```

Comment:
```suggestion
            .contains('5')
```
This applies to more places in this file